src/SettingsPage.tsx

```diff
@@ -90,7 +90,8 @@ export const SettingsPage: React.FunctionComponent<{}> = () => {
             To clone this source repository: git clone
             https://github.com/microsoft/react-native-gallery
           </Text>
-          <Text style={styles.text}>Version: 1.0.0.0</Text>
+          <Text style={styles.text}>Version: 1.0.5.0</Text>
```

Comment:
Yea I came across that documentation and started to use the method. Ran into some issues trying to format and pass that information to the js. Could take another look into it, but regardless the package.json needs manual update, maybe better to have the version displayed in the app connected to the package.json to force users to update that file with the latest number. Otherwise, it runs the risk of being very out of date.

Reply:
Ok just updated to pull version from native code!